docs/data/migration/migration-pickers-v6/migration-pickers-v6.md

```diff
@@ -268,6 +270,33 @@ You can learn more about the new approach on the [dedicated doc page](https://mu
  </LocalizationProvider>
 ```
 
+### Extend `dayjs` in the Day.js adapter
+
+The calls to `dayjs.extend()` to apply the plugins got moved from the file level to the `AdapterDayjs` constructor.
+In case `dayjs` (with its extensions) is needed before the Adapter gets initialized the dayjs instance needs to be
+extended manually with the required plugins first.
+
+```tsx
+import dayjs from 'dayjs';
+import customParseFormatPlugin from 'dayjs/plugin/customParseFormat';
+
+dayjs.extend(customParseFormatPlugin);
+```
+
+The plugin that got moved to the constructor is:
+
+- customParseFormat ('dayjs/plugin/customParseFormat')
+
+The other plugins will (still) be initialized before adapter initialization:
+
+- weekOfYear ('dayjs/plugin/weekOfYear')
+- isBetween ('dayjs/plugin/isBetween')
+- localizedFormat ('dayjs/plugin/localizedFormat')
+
+In the case of plugins that accept options (e.g. the `customParseFormat` plugin) it allows the user to pass custom
+options to the plugins where needed. Currently it is only one plugin that needs and support this feature, but to be
```

Comment:
I do not understand the last sentence

Reply:
this section is not really needed and upon reading it again we can totally live without it.